From: Diego Escalante Urrelo <diegoe@gnome.org>
Date: Thu, 23 Jul 2020 10:30:32 -0500
Subject: wl: Fix misleading indentation
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Mixed spaces and tabs caused a misleading indentation warning.
Tidy up to suppress the warning:

src/wl/sys/wl_linux.c:788:2: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
  788 |  if ((val & 0x0000ff00) != 0)
      |  ^~
src/wl/sys/wl_linux.c:790:3: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
  790 |   bar1_size = pci_resource_len(pdev, 2);
      |   ^~~~~~~~~
---
 src/wl/sys/wl_linux.c | 20 +++++++++++---------
 1 file changed, 11 insertions(+), 9 deletions(-)

diff --git a/src/wl/sys/wl_linux.c b/src/wl/sys/wl_linux.c
index acdac06..a63cc4d 100644
--- a/src/wl/sys/wl_linux.c
+++ b/src/wl/sys/wl_linux.c
@@ -788,16 +788,18 @@ wl_pci_probe(struct pci_dev *pdev, const struct pci_device_id *ent)
 	pci_read_config_dword(pdev, 0x40, &val);
 	if ((val & 0x0000ff00) != 0)
 		pci_write_config_dword(pdev, 0x40, val & 0xffff00ff);
-		bar1_size = pci_resource_len(pdev, 2);
-		#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)
-		bar1_addr = (uchar *)ioremap(pci_resource_start(pdev, 2),
-			bar1_size);
-		#else
-		bar1_addr = (uchar *)ioremap_nocache(pci_resource_start(pdev, 2),
-			bar1_size);
-		#endif /* LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0) */
+
+	bar1_size = pci_resource_len(pdev, 2);
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)
+	bar1_addr = (uchar *)ioremap(pci_resource_start(pdev, 2),
+				     bar1_size);
+#else
+	bar1_addr = (uchar *)ioremap_nocache(pci_resource_start(pdev, 2),
+					     bar1_size);
+#endif /* LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0) */
+
 	wl = wl_attach(pdev->vendor, pdev->device, pci_resource_start(pdev, 0), PCI_BUS, pdev,
-		pdev->irq, bar1_addr, bar1_size);
+		       pdev->irq, bar1_addr, bar1_size);
 
 	if (!wl)
 		return -ENODEV;
